package.json

```diff
@@ -15,14 +15,17 @@
     "@react-navigation/drawer": "^6.1.8",
     "@react-navigation/native": "^6.0.6",
     "@react-navigation/stack": "^6.0.11",
+    "axios": "^0.25.0",
     "react": "17.0.2",
     "react-hook-form": "^7.22.3",
     "react-native": "0.65.1",
+    "react-native-event-listeners": "^1.0.7",
```

Comment:
you don't have to use **react-native-event-listeners** to update the theme. You can directly update the theme from another file and update it in redux, all other variables in different screens using useSeletor Hook will automatically updated.